src/content/tooling/wrangler/kv_commands.md

```diff
@@ -154,6 +156,8 @@ kv-namespaces = [
 ]
 ```
 
+<span id="kv-list"></span>
```

Comment:
should we wrap these spans around the ?

```
<span id="kv-list">
### `list`
</span>
```

I'm not sure but this might be a bit better a11y-wise?